setup-master.yml

```diff
@@ -58,6 +58,21 @@
         src: nagios-otrs.cfg
         dest: /etc/nagios/conf.local.d/check_otrs.cfg
 
+    - name: Menueeintrag fuer Access benutzerabhaengig erstellen
+      template:
+        src: access_2013.desktop
+        dest: /home/{{ item }}/.local/share/applications/access_2013.desktop
+      with_items:
+        - thomasn
+#
```

Comment:
Da das /home Verzeichnis gemounted ist
```
delegate_to: ucs-master....
```

Reply:
Blödsinn, ist ja schon in setup-master.
ggf. noch permissions für den User setzen. Was passiert wenn Verzeichnis nicht exisistiert?